wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/DeferredCommand.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.wpilibj2.command;
+
+import static edu.wpi.first.util.ErrorMessages.requireNonNullParam;
+
+import edu.wpi.first.util.sendable.SendableBuilder;
+import java.util.Set;
+import java.util.function.Supplier;
+
+/**
+ * Runs the command returned by the supplier when this command is initialized, and ends when it
+ * ends. Useful for performing runtime tasks before creating a new command. If this command is
```

Comment:
I'm not sure about this sentence. Maybe replace with something more focused on deferred initialization?

Reply:
Let me know if that's any better. I'm not sure how to best word it.